app/notebook/server/WebSockWrapper.scala

```diff
@@ -1,37 +1,40 @@
-package notebook
-package server
+package notebook.server
 
 import java.util.UUID
 
+import notebook.util.Logging
 import play.api.libs.iteratee._
 import play.api.libs.json._
 
-import notebook.util.Logging
-
 /**
  * Author: Ken
  */
 
 trait WebSockWrapper {
-  def session:String
-  def send(header: JsValue, session: String, msgType: String, channel:String, content: JsValue)
+
+  def session: String
+
+  def send(header: JsValue, session: String, msgType: String, channel: String, content: JsValue)
 }
 
-class WebSockWrapperImpl(sock: Concurrent.Channel[JsValue], val session:String) extends WebSockWrapper with Logging {
+class WebSockWrapperImpl(sock: Concurrent.Channel[JsValue],
+  val session: String) extends WebSockWrapper with Logging {
   private def send(msg: JsValue) {
     logTrace("Sending " + msg)
     sock.push(msg)
   }
 
   val sessionTransformer = (__ \ 'session).json.update(
-    Reads.of[JsString].map{ case JsString(s) => JsString(session) }
+    Reads.of[JsString].map { case JsString(s) => JsString(session) }
   )
-  def injectCurrentSession(header:JsValue):JsValue = {
+
+  def injectCurrentSession(header: JsValue): JsValue = {
     val newHeader = header.transform(sessionTransformer)
     newHeader.get
   }
 
-  def send(header: JsValue, __session: String /*ignored in favor of session_id!!!*/, msgType: String, channel:String, content: JsValue) {
+  def send(header: JsValue, __session: String /*ignored in favor of session_id!!!*/ ,
```

Comment:
consider a space between comment and the comment delimiters /* and */ splitting out parameters to their own line if needed for horizontal space. Just a thought.